imagersite/imager_images/models.py

```diff
@@ -0,0 +1,75 @@
+"""Model photos and albums for database items."""
+
+from django.db import models
+from django.conf import settings
+from django.utils.encoding import python_2_unicode_compatible
+
+# Create your models here.
+
+PUBLISHED_CHOICES = [('private', 'private'), ('shared', 'shared'), ('public', 'public')]
+PUBLISHED_DEFAULT = PUBLISHED_CHOICES[0]
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    """Model a single photo in database."""
+
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='photos')
+    albums = models.ManyToManyField('Album', related_name='photos')
```

Comment:
When using a string declaration for the model in an M2M relationship, your strings should be formatted as `'app_name.model_name'`.